grist-deployment-tests/test/e2e/test.e2e.js

```diff
@@ -0,0 +1,30 @@
+import { expect } from '@wdio/globals'
+import LoginPage from '../pageobjects/login.page.js'
+import HomePage from '../pageobjects/homepage.page.js'
+import assert from 'assert';
+
+describe('Login', () => {
+  it('should login and logout with valid credentials using Agents Connect', async () => {
```

Comment:
I keep suggesting a change here :)
I think it's more explicit as :

* AgentConnect no longer exists
* OIDC helps non .gouv devs to understand what we are talking about. 

```suggestion
  it('should login and logout with valid credentials using ProConnect OIDC', async () => {
```

Reply:
You're right, I missed that, thanks for the suggestion. Done! :)